implementation/src/main/java/io/smallrye/config/utils/ConfigSourceUtil.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.smallrye.config.utils;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+/**
+ * utilities and constants for {@link ConfigSource} implementations
+ *
+ * @author helloween
+ */
+public class ConfigSourceUtil {
+    public static final String CONFIG_ORDINAL_KEY = "config_ordinal";
+    public static final String CONFIG_ORDINAL_DEFAULT_VALUE = "100";
```

Comment:
I think the name should include the ordinal value

Reply:
CONFIG_ORDINAL_DEFAULT_VALUE_100?

Reply:
Maybe `CONFIG_ORDINAL_100`?

Reply:
ok